GiniCapture/Classes/Core/Screens/Document picker/Gallery/AlbumsPickerViewController.swift

```diff
@@ -117,7 +118,7 @@ extension AlbumsPickerViewController: UITableViewDataSource {
                 let frame: CGRect = tableView.frame
                 let buttonTitle = NSLocalizedStringPreferredFormat("ginicapture.albums.selectMorePhotosButton",
                                                                    comment: "Title for select more photos button")
-                let selectButton = UIButton(frame: CGRect(x: frame.size.width - 250, y: 0, width: 250, height: headerHeight))
+                let selectButton = UIButton(frame: CGRect(x: frame.size.width - selectButtonWidth, y: 0, width: selectButtonWidth, height: headerHeight))
```

Comment:
Could the button have a minimum width and grow to fit the title? I am asking in case clients set a title which doesn't fit in the current width. Also what happens when text size is increased in the accessibility settings?

Reply:
i will add a xib for header